packages/web-react/src/types/fieldGroup.ts

```diff
@@ -0,0 +1,14 @@
+import { HelperTextProps, SpiritFieldGroupElementPropsWithRef, Validation, ValidationTextType } from './shared';
+import { LabelProps } from './label';
```

Comment:
this also sort by abc pls, label import first and then shared.

Reply:
done